### PR TITLE
Project

### PR DESCRIPTION
**Pull request by AndreLau11**
Add the modules to my previous project.

I'm just removing the node_modules project - please check it out :

https://github.com/AndreLau11/Awesome_Books_Modules/pull/1

### CODE REVIEW

File: modules/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+function time() {
+  const siteDate = document.querySelector('#date');
+  siteDate.textContent = DateTime.now().toLocaleString({
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: '2-digit',
+  });
+}
```
Comment (thecodechaser) [inline modules/index.js:81]: - [ ] [optional]Please change this function to the arrow function this will make sure the code follows ES6 syntax.

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @AndreLau11👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Optional suggestions
- [ ] check the comments under the review

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/index.js (hunk 1 above)
Comment (FrederickMih) [inline modules/index.js:81]: - [Optional] Please no function keyword should be seen in this particular project(Awesome books: with ES6)

File: modules/nav-items.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const bookList = document.querySelector('.book-list');
+const header = document.querySelector('.header');
+const newBook = document.querySelector('.add-new');
+const contact = document.querySelector('.contact-section');
+const titleH1 = document.querySelector('h1');
+const navItems = Array.from(document.querySelectorAll('.navItems')[0].children);
+
+export default function navigate(key) {
```
Comment (FrederickMih) [inline modules/nav-items.js:8]: - [Optional] Please no function keyword should be seen in this particular project(Awesome books: with ES6). All functions should be rather arrow functions

**Review by FrederickMih — APPROVED** [review]
Hi @AndreLau11 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**